pkg/config/user_config.go

```diff
@@ -99,9 +99,10 @@ type GitConfig struct {
 }
 
 type PagingConfig struct {
-	ColorArg  string `yaml:"colorArg"`
-	Pager     string `yaml:"pager"`
-	UseConfig bool   `yaml:"useConfig"`
+	ColorArg            string `yaml:"colorArg"`
+	Pager               string `yaml:"pager"`
+	UseConfig           bool   `yaml:"useConfig"`
+	ExternalDiffCommand string `yaml:"externalDiffCommand"`
```

Comment:
I wasn't sure if this should go as a subkey of `pager` or outside of it. It's not technically related to pagers, but from a user's point of view it's conceptually similar. Also, if we later decide to have an array of pagers so that you can cycle through them with a keybinding, having it inside of `pager` will make this easier.

Reply:
I'm happy with this